R/twobit.R

```diff
@@ -118,15 +115,16 @@ setMethod("import", "TwoBitFile",
                    ...)
           {
             lkup <- get_seqtype_conversion_lookup("B", "DNA")
-            sl <- .seqlengths_TwoBitFile(con)
+            sl <- Rtwobitlib::twobit_seqlengths(path(con))
             sn <- extractROWS(names(sl), match(seqnames(which), seqlevels(con)))
             if (any(is.na(sn))) {
                 stop("'seqnames' not in 2bit file: ",
                      paste0("'", unique(seqnames(which)[is.na(sn)]), "'",
                             collapse=", "))
             }
-            ans <- .Call(TwoBitFile_read, twoBitPath(path(con)),
-                         sn, as(ranges(which), "IRanges"), lkup)
```

Comment:
@hpages 
I wasn't sure what to do with `lkup` object here. I'm not sure what it does.